.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 coverage/
+*.lock
```

Comment:
Is this for yarn?

Reply:
yes, I just noticed that npm's lock file is package-lock.json.

Reply:
I removed it. I read it is useful [from this blog](https://jpospisil.com/2017/06/02/understanding-lock-files-in-npm-5.html)